### DISCUSSION TITLE
HTTP 429 (Too many requests) from panasonic-comfort-cloud

### DISCUSSION THREAD

**Discussion by majakmarcin** [bisand/node-red-contrib-panasonic-comfort-cloud#25] [Q&A]
Does anyone have the same problem? What is the maximum number of requests per minute? How long do I have to wait until I will be able to send requests again?

**Comment by bisand** [comment]
Yes, we have the same problem. I don’t know the exact request rate, but Panasonic have a cap on how many requests you can send per minute/hour/day. This is very anoying, and there is nothing we can do other than complain directly to Panasonic to have them raise the restrictions. If enough of us do that, they might listen…

**Comment by smadds** [comment]
I'm still having issues with this. I thought that rate limiting would sort it, but even spacing 5mins between calls did not prevent triggering.
I wonder if it would help to log out between messages? I've had a scan of the panasonic-comfort-cloud-api, but can't see mention of a logout command. Is this something you could try adding, @bisand ?

**Comment by bisand** [comment]
Is this still an issue? The underlying Panasonic library has now been replaced.
